### DISCUSSION TITLE
Who is Mich?

### DISCUSSION THREAD

**Discussion by diyoyo** [immich-app/immich#5389] [General]
What does Immich mean? "I am Mich" ? "In Memoriam Mich" ? "Image->Imache->Immich" ? "Image Mich "?

So Who is/was Mich?

**Comment by alextran1502** [comment]
Image 😉

**Comment by etnoy** [comment]
Once upon a digital landscape, in the vast realms of code and creativity, there existed a mysterious figure named Mich. Mich was a coding wizard, a digital sorcerer who lived in the shadows, weaving lines of logic and spells of syntax to bring forth a creation that would change the way people managed their memories.

Legend has it that Mich, driven by a passion for photography and a desire for privacy, embarked on a quest to create the ultimate self-hosted open-source photo manager. The result of Mich's relentless efforts was none other than Immich – a magical repository for memories, a sanctuary where pixels transformed into nostalgic fragments of the past.

The tale of Immich spread far and wide, whispered among programmers and photographers alike. The legend had it that when Mich completed the final lines of code, a resounding voice echoed through the digital valleys, declaring, "I'm Mich," as if the very essence of the creator became intertwined with the creation.

Immich was a masterpiece, a symphony of elegant design and functionality. It allowed users to host their photo collections on their terms, embracing the philosophy of "I'm Mich" – a declaration of individuality, autonomy, and control over one's digital memories. Users across the realms of the internet flocked to Immich, drawn by its allure and the promise of a photo management utopia.

In the user interface, subtle references to "I'm Mich" were sprinkled like magical breadcrumbs, inviting users to explore the depths of their own digital narratives. The login screen bore the inscription, "Welcome to Immich – where every pixel says, 'I'm Mich.'" It wasn't just a photo manager; it was a declaration of identity, a proclamation of the creator's presence in every corner of the digital universe.

Immich's logo, a stylized silhouette of Mich with a camera, became a symbol of the photo management revolution. Mich's enigmatic persona lingered in the code, a ghostly guardian ensuring the sanctity of memories and the adherence to the principles of self-hosted freedom.

As Immich spread its wings, Mich remained an elusive figure, rarely seen but always felt in the digital air. The community around Immich grew stronger, united by the shared belief in the power of personal control over their memories. Users would jest, saying, "Every time I organize my photos, I feel like I'm channeling my inner Mich."

The legend of Mich and Immich continued to evolve, a saga woven into the tapestry of the digital age. Whether you were a seasoned coder, a photography enthusiast, or a casual user, the name "Mich" became synonymous with the spirit of independence and the assertion, "I'm Mich," echoed in every click, every upload, and every pixel stored within the realms of Immich.

**Reply by diyoyo** [reply]
I guess credits go to ChatGPT? 
Nice one.

**Reply by etnoy** [reply]
Yeah, I would never have spent time writing that thing myself 😅

**Reply by diyoyo** [reply]
42 minutes ago...
